packages/storybook/src/web-components/link/link.stories.tsx

```diff
@@ -0,0 +1,187 @@
+import tokens from '@lux-design-system/design-tokens/dist/index.json';
+import {
+  LuxLink as Link,
+  LuxDocument,
+  LuxIconChevronLeft,
+  LuxParagraph,
+} from '@lux-design-system/web-components-react';
+import type { JSX } from '@lux-design-system/web-components-stencil';
+import type { Meta, StoryObj } from '@storybook/react';
+import { type PropsWithChildren } from 'react';
+import tokensDefinition from './tokens.json';
+import { VisualStates } from './visual/States';
+import { createDesignTokensStory, createVisualRegressionStory, VisualRegressionWrapper } from '../../utils';
+
+const href = 'http://logius.nl/';
+
+const LuxLink = (props: PropsWithChildren<JSX.LuxLink>) => <Link {...props} />;
+
+type Story = StoryObj<typeof meta>;
+
+const meta = {
+  title: 'Web Components/Link',
+  id: 'web-components-link',
+  component: LuxLink,
+  decorators: [
+    (Story) => (
+      <LuxDocument>
```

Comment:
Als dit een context is waarin de `lux-link` altijd moet leven moeten we dat welk duidelijk maken. Ergens expliciet benoemen

Reply:
Alle componenten zouden hierin moeten leven, die font-settings inherit. Bij de Button zagen we het niet omdat Aline de font-family-token had gezet.

Reply:
Dan meoten we dat wel ergens gaan documenteren. Bijvoorbeeld in de Installatiehandleiding